lib/landrush/cap/linux/read_host_visible_ip_address.rb

```diff
@@ -39,7 +39,7 @@ def self.read_host_visible_ip_address(machine)
         end
 
         def self.command
-          %Q(hostname -I)
+          %Q(hostname -I 2>/dev/null || /sbin/ip ad | sed -ne 's/^\\W*inet \\([0-9.]*\\).* global .*$/\\1/p' 2>/dev/null)
```

Comment:
This PR can be considered complimentary to the binary solution proposed in #193.

The `ip` utility was mentioned [here](https://github.com/vagrant-landrush/landrush/pull/193#issuecomment-225037763), but I dont see any information designating it as an unsuitable for this purpose. According to the man page it was added alongside linux 2.2 in 1999.  On Linux, `ip` is the long-term replacement for `ifconfig`, so is a better option in all respects.